subworkflows/local/bam_impute_quilt/tests/main.nf.test.snap

```diff
@@ -62,22 +62,22 @@
                         "id": "NA12878",
                         "panel": "1000GP",
                         "chr": "22",
-                        "chunk": "16570065-16597215",
+                        "chunk": "22:16570065-16597215",
                         "tools": "quilt"
                     },
-                    "NA12878_22_16570065-16597215.impute.annotate.vcf.gz",
-                    "NA12878_22_16570065-16597215.impute.annotate.vcf.gz.tbi"
+                    "NA12878_22_22:16570065-16597215.impute.annotate.vcf.gz",
```

Comment:
I think adding ":" in filenames might be problematic.

Reply:
Fixed it